### ISSUE TITLE
Make sure ontologies release JSON

### ISSUE THREAD

**Issue by matentzn** [INCATools/ontology-development-kit#267]
List provided by @sierra-moxon

| Ontology | JSON |
| ---- | ---- |
| GO | [JSON (subsets)](http://geneontology.org/docs/download-ontology/) | 
| SO | - | 
| DOID | - | 
| MI | - | 
| ZFA | - | 
| ZFS | - | 
| WBBT | [JSON](http://purl.obolibrary.org/obo/wbbt/wbbt.json) | 
| CL | [JSON](http://purl.obolibrary.org/obo/cl/cl.json) | 
| FBBT | [JSON](http://purl.obolibrary.org/obo/fbbt/fbbt.json) | 
| FBCV | [JSON](http://purl.obolibrary.org/obo/fbcv/fbcv.json) | 
| MMUSDV | - | 
| BSPO | - | 
| MMO | - | 
| WBLS | [JSON](http://purl.obolibrary.org/obo/wbls/wbls.json) | 
| EMAPA | - | 
| MA | - | 
| ECO | - | 
| UBERON | - | 
| ECOMAP | - | 
| PATO | [JSON](http://purl.obolibrary.org/obo/pato/pato.json) | 
| MPATH | - | 
| CHEBI | - | 
| UBERON (basic.obo) | - | 
| ZECO | [JSON](http://purl.obolibrary.org/obo/zeco/zeco.json) |

**Comment by matentzn** [comment]
As this will probably take a couple of years @sierra-moxon: would you be interested in simpy developing a quick pipeline that pulls the OWL of all those ontologies and transform them into OBO-JSON?

**Comment by cmungall** [comment]
why would this take years? Most should be a few minutes work, others with legacy pipelines, e.g. chebi, would be harder

**Comment by cmungall** [comment]
what is ecomap?

**Comment by cmungall** [comment]
@sierra-moxon, not sure you need both MA and EMAPA, I think just EMAPA?

Much as I love BSPO, don't think you need it?

There is also this list:
https://www.alliancegenome.org/privacy-warranty-licensing

but is this canonical? where is EDAM used?

**Comment by sierra-moxon** [comment]
The link above: https://www.alliancegenome.org/privacy-warranty-licensing is as canonical as we have at the moment. :) It is updated frequently and attempts to be transparent about the ontologies that groups use and whether or not we have use cases for them at the Alliance (yet).  EDAM is used in some form by SGD, but has not been brought into the Alliance yet.  (I imagine it will be at some point).  

BSPO is still used (perhaps only by ZFIN), to annotation expression and phenotype.  We do submit BSPO tagged annotations to the Alliance.  

I can check on EMAPA/MA.  Thanks for pointing this out.

**Comment by matentzn** [comment]
@cmungall yeah the guestimate was more for migrating all pipelines to a proper ODK like setup; having a JSON export is not as bad.. Unless there is no pipeline at all, and no one really maintaining the ontology of course.

**Comment by gouttegd** [comment]
Not sure why this is here. It is trivial to produce and release JSON-formatted artefacts with the ODK (just add `json` to the `export_formats` ODK configuration option).

If there are ontologies out there that are not doing that, it’s not the ODK’s fault.
